versioned_docs/version-v1.4/integrations/integrations.md

```diff
@@ -0,0 +1,25 @@
+---
```

Comment:
how about add Chinese version of this docs?

Reply:
@rambohe-ch Chinese version has been commited.